NBXplorer/Repository.cs

```diff
@@ -589,7 +588,7 @@ private void RefillAvailable(DBreeze.Transactions.Transaction tx, DerivationStra
 					Redeem = derivation.Redeem,
 					DerivationStrategy = strategy,
 					Feature = derivationFeature,
-					KeyPath = DerivationStrategyBase.GetKeyPath(derivationFeature).Derive(index, false)
+					KeyPath = DerivationFeature.Direct == derivationFeature ? new KeyPath(index.ToString()) : DerivationStrategyBase.GetKeyPath(derivationFeature).Derive(index, false)
```

Comment:
why not just 'DerivationStrategyBase.GetKeyPath(derivationFeature).Derive(index, false)' ?

Reply:
Yes, Correct , actually changed the GetKeyPath() function later.
DerivationStrategyBase.GetKeyPath(derivationFeature).Derive(index, false) == new KeyPath().Derive(1)